package.json

```diff
@@ -419,6 +425,7 @@
     "embedding-sdk:generate-nextjs-compat": "tsx enterprise/frontend/src/embedding-sdk/bin/generate-nextjs-compat.ts",
     "embedding-sdk:test-unit": "yarn test-unit enterprise/frontend/src/embedding-sdk/",
     "embedding-sdk:tsc": "tsc --project tsconfig.sdk.json",
+    "embedding-sdk:docs:generate": "typedoc --options ./.typedoc/typedoc.config.mjs",
```

Comment:
We should run this command after `yarn build-embedding-sdk` to use the latest `dist`
I can add `yarn build-embedding-sdk` as the part of this command

Also this command should be run every time we want to regenerate documentation